bundle/config/mutator/paths/pipeline_paths_visitor.go

```diff
@@ -44,6 +70,10 @@ func VisitPipelinePaths(value dyn.Value, fn VisitFunc) (dyn.Value, error) {
 
 	for _, rewritePattern := range pipelineRewritePatterns() {
 		newValue, err = dyn.MapByPattern(newValue, rewritePattern.pattern, func(p dyn.Path, v dyn.Value) (dyn.Value, error) {
+			if rewritePattern.skipRewrite(v.MustString()) {
```

Comment:
Let's avoid MustString() and check for type explicitly -- unclear from the code that it is safe.